### ISSUE TITLE
Captchas not working

### ISSUE THREAD

**Issue by digitalnomad91** [FlareSolverr/FlareSolverr#811]
### Have you checked our README?

- [X] I have checked the README

### Have you followed our Troubleshooting?

- [X] I have followed your Troubleshooting

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: Latest
- Last working FlareSolverr version: Latest
- Operating system: Debian 11
- Are you using Docker: [yes/no] yes
- FlareSolverr User-Agent (see log traces or / endpoint):  Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36
- Are you using a VPN: [yes/no] no
- Are you using a Proxy: [yes/no] no
- Are you using Captcha Solver: [yes/no] yes
- If using captcha solver, which one: CloudFlare? Not sure.
- URL to test this issue: http://bscscan.com/contractsVerified
```

### Description

So noticed that flaresolverr stopped working and was failing to solve all CF captchas for my request to http://bscscan.com/contractsVerified. They all just time out and then I get a 500 error. Happens on every machine I've tried, updated to latest rls. Noticed on README that the CAPTCHA is broken, so I suppose that's what is going on. I always knew this was a war of attrition, an arms race of sorts. Any ideas on what the next move is guys? Where can I get started if I want to try and help with the next step to defeat whatever they changed? 

### Logged Error Messages

```text
2023-07-11 20:32:49 INFO     Serving on http://0.0.0.0:8191
2023-07-11 20:33:11 INFO     Incoming request => POST /v1 body: {'cmd': 'request.get', 'url': 'http://bscscan.com/contractsVerified', 'maxTimeout': 60000}
2023-07-11 20:33:12 INFO     Challenge detected. Title found: Just a moment...
2023-07-11 20:34:12 ERROR    Error: Error solving the challenge. Timeout after 60.0 seconds.
2023-07-11 20:34:12 INFO     Response in 60.808 s
2023-07-11 20:34:12 INFO     172.17.0.1 POST http://localhost:8191/v1 500 Internal Server Error
```

**Comment by Hyperz** [comment]
Same issue here. CF just shows an infinite amount of turnstiles in the challenges. Looks like it might be fingerprinting since it doesn't happen outside of Selenium.

**Comment by digitalnomad91** [comment]
> Same issue here. CF just shows an infinite amount of turnstiles in the challenges. Looks like it might be fingerprinting since it doesn't happen outside of Selenium.

Yep, they definitely updated something. Hopefully we can get a resolution soon. Also, I just realized the warning in the README.md was from a while ago, so not related to this issue at all.

**Comment by digitalnomad91** [comment]
Got some more info when ran in debug log mode:  
  
`2023-07-11 20:48:43 INFO     ReqId 140257445910272 Incoming request => POST /v1 body: {'cmd': 'request.get', 'url': 'http://bscscan.com/contractsVerified', 'maxTimeout': 60000}`  
`2023-07-11 20:48:43 DEBUG    ReqId 140257445910272 Launching web browser...`  
``2023-07-11 20:48:43 DEBUG    ReqId 140257445910272 Started executable: `/app/chromedriver` in a child process with pid: 178``  
`2023-07-11 20:48:43 DEBUG    ReqId 140257445910272 New instance of webdriver has been created to perform the request`  
`2023-07-11 20:48:43 DEBUG    ReqId 140257412339456 Navigating to... http://bscscan.com/contractsVerified`  
`2023-07-11 20:48:44 INFO     ReqId 140257412339456 Challenge detected. Title found: Just a moment...`  
`2023-07-11 20:48:44 DEBUG    ReqId 140257412339456 Waiting for title (attempt 1): Just a moment...`  
`2023-07-11 20:48:54 DEBUG    ReqId 140257412339456 Timeout waiting for selector`  
`2023-07-11 20:48:54 DEBUG    ReqId 140257412339456 Try to find the Cloudflare verify checkbox`  
`2023-07-11 20:48:54 DEBUG    ReqId 140257412339456 Cloudflare verify checkbox not found on the page`  
`2023-07-11 20:48:54 DEBUG    ReqId 140257412339456 Try to find the Cloudflare 'Verify you are human' button`  
`2023-07-11 20:48:54 DEBUG    ReqId 140257412339456 The Cloudflare 'Verify you are human' button not found on the page`  
`2023-07-11 20:48:56 DEBUG    ReqId 140257412339456 Waiting for title (attempt 2): Just a moment...`  
`2023-07-11 20:49:07 DEBUG    ReqId 140257412339456 Timeout waiting for selector`  
`2023-07-11 20:49:07 DEBUG    ReqId 140257412339456 Try to find the Cloudflare verify checkbox`  
`2023-07-11 20:49:07 DEBUG    ReqId 140257412339456 Cloudflare verify checkbox not found on the page`  
`2023-07-11 20:49:07 DEBUG    ReqId 140257412339456 Try to find the Cloudflare 'Verify you are human' button`  
`2023-07-11 20:49:07 DEBUG    ReqId 140257412339456 The Cloudflare 'Verify you are human' button not found on the page`  
`2023-07-11 20:49:09 DEBUG    ReqId 140257412339456 Waiting for title (attempt 3): Just a moment...`  
`2023-07-11 20:49:19 DEBUG    ReqId 140257412339456 Timeout waiting for selector`  
`2023-07-11 20:49:19 DEBUG    ReqId 140257412339456 Try to find the Cloudflare verify checkbox`  
`2023-07-11 20:49:19 DEBUG    ReqId 140257412339456 Cloudflare verify checkbox not found on the page`  
`2023-07-11 20:49:19 DEBUG    ReqId 140257412339456 Try to find the Cloudflare 'Verify you are human' button`  
`2023-07-11 20:49:19 DEBUG    ReqId 140257412339456 The Cloudflare 'Verify you are human' button not found on the page`  
`2023-07-11 20:49:21 DEBUG    ReqId 140257412339456 Waiting for title (attempt 4): Just a moment...`  
`2023-07-11 20:49:31 DEBUG    ReqId 140257412339456 Timeout waiting for selector`  
`2023-07-11 20:49:31 DEBUG    ReqId 140257412339456 Try to find the Cloudflare verify checkbox`  
`2023-07-11 20:49:31 DEBUG    ReqId 140257412339456 Cloudflare verify checkbox not found on the page`  
`2023-07-11 20:49:31 DEBUG    ReqId 140257412339456 Try to find the Cloudflare 'Verify you are human' button`  
`2023-07-11 20:49:31 DEBUG    ReqId 140257412339456 The Cloudflare 'Verify you are human' button not found on the page`  
`2023-07-11 20:49:33 DEBUG    ReqId 140257412339456 Waiting for title (attempt 5): Just a moment...`  
`2023-07-11 20:49:43 DEBUG    ReqId 140257445910272 A used instance of webdriver has been destroyed`  
`2023-07-11 20:49:43 ERROR    ReqId 140257445910272 Error: Error solving the challenge. Timeout after 60.0 seconds.`  
`2023-07-11 20:49:43 DEBUG    ReqId 140257445910272 Response => POST /v1 body: {'status': 'error', 'message': 'Error: Error solving the challenge. Timeout after 60.0 seconds.', 'startTimestamp': 1689108523198, 'endTimestamp': 1689108583828, 'version': '3.2.1'}`  
`2023-07-11 20:49:43 INFO     ReqId 140257445910272 Response in 60.63 s`  
`2023-07-11 20:49:43 INFO     ReqId 140257445910272 172.17.0.1 POST http://localhost:8191/v1 500 Internal Server Error`

**Comment by gingerbeardman** [comment]
> ultrafunkamsterdam/undetected-chromedriver#1388

There's a workaround in here (open dev tools on the tab)

**Comment by digitalnomad91** [comment]
Yep this workaround works, hopefully it lasts for a while and Flaresolverr and that other Python tool can implement a more long term solution. So far I haven't run into any issues using IP's from my datacentre provider or from making too many requests, but I may have to implement a bunch of these on different IPs and have a round-robin system or something. Idk, this is going to be a problem again, and again, and then again. It's an arms race / war of attrition for sure. Anyways, thanks again guys and if there's anything I can do to help implement a solution in Flaresolverr lmk.   
  
~ Peace

**Comment by gingerbeardman** [comment]
> ultrafunkamsterdam/undetected-chromedriver#1388

2nd workaround in here (open new tab using js, which will work, old tab can then be closed)

**Comment by MrTyton** [comment]
Does that work if flaresolverr is running in headless? Where would we stick that code into?

**Comment by digitalnomad91** [comment]
> > ultrafunkamsterdam/undetected-chromedriver#1388
> 
> 2nd workaround in here (open new tab using js, which will work, old tab can then be closed)
> https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/1388#issuecomment-1632000503
> 
> Q: how to easily get either of these in my install?

I haven't tried this one yet as just adding the dev tools option worked for me (in headless mode also btw), but it's probably similar and needs to be added in sec/utils.py.
 
I made a PR with that workaround, so you can pull that branch if you want, but it's also just adding a single LOC so not difficult to do yourself.

I'll try to put together another branch and PR later today with the 2nd workaround as well. 

I didn't setup a docker image / container command for the workaround so you'll either have to create your own dockerfile and build your own custom image, or just run python3 src/Flaresolverr.py (either in a tmux/screen session so it stays running after your SSH session ends, or maybe create a systemd service if you're using a distro with it). 

If I get a chance I'll put together a dockerfile and some example commands on building a new image / launching a container with the patches(s). And if I get a chance I'll also put together a systemd service file with the few commands it would take for building your own patched image and launching a container. 

You could also just run the python3 command in detached mode and maybe redirect the output to a log file so it's easy to just tail and see what it's doing. 

From what I've seen there may be issues with the first one line workaround on Windows, but so far it has been working well for me. 

Anyway, good luck and I'll try to comment again or make another PR with the other patch and/or with the custom docket image build / deploy and perhaps a sample systemd service file as well. 

Best Regards,
Andrew Corbin

**Comment by digitalnomad91** [comment]
> Does that work if flaresolverr is running in headless? Where would we stick that code into?

It has been working for me in headless mode, but I believe some others may have been having issues, particularly running in Windows. I'd highly recommend using WSL and Debian if you have to use Windows though. Tons of stuff always seems to be a bit wonky or come with annoying caveats when running directly in Doze. 

Read my last comment for ways to use the first patch, and I'm pretty sure the second one would just involve adding those LOC in the same file but with proper variable names. I'll try to make another PR and a dockerfile and/or systemd file as well, but it's also easy enough to just run it in tmux or in a detached mode that redirects the output to a log file you can tail. 

Hope that helps and best of luck! 

~ Peace ✌️

**Comment by MrTyton** [comment]
PR would be easiest. I'm running it in an Unraid Docker headless, so while I can made modifications if the container restarts then I lose them at some point lol. I'll give your fixes a manual go and see how it does in the meanwhile. Thanks!!

**Comment by gingerbeardman** [comment]
> Q: how to easily get either of these in my install?

https://github.com/FlareSolverr/FlareSolverr/pull/812

This worked for me.

**Comment by thags** [comment]
Just the dev tools was not enough to get around this issue when I tested. Opened [this PR](https://github.com/FlareSolverr/FlareSolverr/pull/816) which seems to resolve it for me, running on windows.

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/releases/v3.2.2

Linux/Docker version working, Windows version throwing error:

```text
[1008] Module object for pyimod02_importers is NULL!
Traceback (most recent call last):
  File "PyInstaller\loader\pyimod02_importers.py", line 22, in <module>
  File "pathlib.py", line 14, in <module>
  File "urllib\parse.py", line 40, in <module>
ModuleNotFoundError: No module named 'ipaddress'
Traceback (most recent call last):
  File "PyInstaller\loader\pyiboot01_bootstrap.py", line 17, in <module>
ModuleNotFoundError: No module named 'pyimod02_importers'
[1008] Failed to execute script 'pyiboot01_bootstrap' due to unhandled exception!
```

I have no idea why and I'm about to fall asleep. I'll have a look at it tomorrow, but if anyone else can spot why and let me know, it'd be much appreciated.

**Comment by ilike2burnthing** [comment]
Windows hotfix now available - https://github.com/FlareSolverr/FlareSolverr/releases/v3.2.2-hotfix

**Comment by thags** [comment]
The merged PR seems to have resolved this for me.

**Comment by ashame** [comment]
@ilike2burnthing getting an identical error running the latest linux build:

```
[67399] Module object for pyimod02_importers is NULL!
Traceback (most recent call last):
  File "PyInstaller/loader/pyimod02_importers.py", line 22, in <module>
  File "pathlib.py", line 14, in <module>
  File "urllib/parse.py", line 40, in <module>
ModuleNotFoundError: No module named 'ipaddress'
Traceback (most recent call last):
  File "PyInstaller/loader/pyiboot01_bootstrap.py", line 17, in <module>
ModuleNotFoundError: No module named 'pyimod02_importers'
```

```
> lsb_release -a
LSB Version:    n/a
Distributor ID: Gentoo
Description:    Gentoo Linux
Release:        2.13
Codename:       n/a
```

```
> uname -r
5.15.112-gentoo-whatbox
```

This is a fresh download of latest `flaresolverr_linux_x64.tar.gz (sha256: edba60d929660ea10bef59afd24dd7d7c60b3284fd6f038eb31a836c29b8fc3d)`.

**Comment by benkrej** [comment]
With todays cloudflare patch our 2 solutions don't seem to work anymore.

**Comment by MrTyton** [comment]
> With todays cloudflare patch our 2 solutions don't seem to work anymore.

Can confirm.

**Comment by puffhub** [comment]
Is anyone resolved this problem? 😭

**Comment by 01smicha** [comment]
I can also confirm I am experiencing the same issue, even after updating docker image to the latest 3.2.2 hotfix.

log from flaresolver:
`2023-07-22 12:17:41 INFO     Incoming request => POST /v1 body: {'maxTimeout': 55000, 'cmd': 'request.get', 'url': 'https://ipt.lol/t?73=&26=&55=&78=&23=&24=&25=&66=&65=&79=&22=&5=&99=&4=&o=time'}
2023-07-22 12:17:45 INFO     Challenge detected. Title found: Just a moment...
2023-07-22 12:18:41 ERROR    Error: Error solving the challenge. Timeout after 55.0 seconds.
2023-07-22 12:18:41 INFO     Response in 60.119 s
2023-07-22 12:18:41 INFO     172.17.0.7 POST http://172.17.0.10:8191/v1 500 Internal Server Error`

log from jackett:
`07-22 13:29:18 Info Torznab search in TorrentGalaxy => Found 50 releases (from cache) [0ms] 
07-22 13:29:36 Info CardigannIndexer(torrentgalaxy): Relogin required 
07-22 13:29:36 Info CardigannIndexer(torrentgalaxy): Relogin required 
07-22 13:29:41 Error CardigannIndexer (torrentgalaxy): Found captcha during automatic login, aborting 
07-22 13:29:41 Error Error on 20230722132941 for TorrentGalaxy.txtSystem.Exception: Relogin failed
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in ./Jackett.Common/Indexers/CardigannIndexer.cs:line 1719`

**Comment by kenanAlhindi97** [comment]
cloudflare on a warpath lol 
can confirm that nothing is working, hotfix or workarounds

**Comment by YasserAntonio** [comment]
I'm using the latest docker  version on an arm server. It seems that the challenges with captcha redirection aren't detected anymore :

```bash
curl -L -X POST 'http://192.168.1.29:8191/v1' \
-H 'Content-Type: application/json' \
--data-raw '{
  "cmd": "request.get",
  "url":"https://dl.opensubtitles.org/en/subtitleserve/sub/9641843",
  "maxTimeout": 60000
}'

{"status": "ok", "message": "Challenge not detected!", "solution": {"url":  
"https://www.opensubtitles.org/en/captcha/redirect- 
%7Cen%7Cdownload%7Csub%7C641843", "status": 200, ...
```

**Comment by digitalnomad91** [comment]
Working implementation of this 3rd patch for undetect\_chromedriver (single python file):  
https://github.com/digitalnomad91/udetect_coudflare_working_patch/

Here is the latest version of flaresolverr with this fix implemented:   
https://github.com/Dexcelerate/FlareSolverr-udetect-patch/

Note: I haven't tested running it through a new docker image / container, but it should work just fine. I'll include a systemd service implementation for those that want to run it standalone.

Still waiting on this (seemingly) more robust method to be implemented & released:  
https://github.com/ultrafunkamsterdam/undetected-chromedriver/discussions/1420

I also put together a small browser extension that can run in Chrome or Firefox that worked well enough to get me through the periods where CF was winning. Will probably be useful for if/when there are no known solutions like right now:  
https://github.com/digitalnomad91/cloudflare-bypass-extension

**Comment by antivirtel** [comment]
Hi @digitalnomad91, I'm glad this is progressing, thanks for the hard work. BTW, the link https://github.com/Dexcelerate/FlareSolverr-udetect-patch/ is broken for me (404). Does that work for you? Thanks

**Comment by digitalnomad91** [comment]
@[**antivirtel**](https://github.com/antivirtel) **My mistake I forgot to update the repo url, should be good now!** 

**https://github.com/digitalnomad91/FlareSolverr-udetect-patch/**

**Comment by Nackophilz** [comment]
@digitalnomad91 why not make a pull request? From what I understand, you've make a fork of this repository, so if its functional the maintener of flaresolverr can just add straight away your changes ?

**Comment by XamYp** [comment]
I agree with that, it would benefit everyone if you could make a pull request from this issue

**Comment by digitalnomad91** [comment]
> @digitalnomad91 why not make a pull request? From what I understand, you've make a fork of this repository, so if its functional the maintener of flaresolverr can just add straight away your changes ?

I tried making a PR the first time and it was deleted. Tried to do one again from a fork but I kept on getting 403 errors and was confused as to why, but I noticed I couldn't even merge my branch into my own repo as it was forcing a PR onto the fork as well.

I'd have been fine with this but I was confused about the 403 error at first and thought it might be related, but it turned out to be my SSH key expired lol. 

Also this is a bit of a hacky patch as I commented out several of the default options and potentially a few other things that the maintainers of the official repo may not necessarily want to have commented out. I made the other repo with the single Python file to try and help show a working method in a way that's easier to understand. 

I think the only changed I made were in src/utils.py where I added that option that makes use of req.url, and I also had to comment out some of the default settings that were there as CF was defeating it with them on. That's probably a clue as to how CF is defeating it btw, they are fingerprinting or something and one of the options that I commented out was giving it away. 

Honestly this is an arms race or a war of attrition, and I won't be surprised if CF does something and breaks this one pretty quickly as well. Hopefully not, but chances are it will eventually happen again. 

The webextension workaround is kind of ugly and could definitely be cleaned up and improved upon, but it's something that I think should more or less always work and can serve as a backup when all else fails.

**Comment by digitalnomad91** [comment]
> I agree with that, it would benefit everyone if you could make a pull request from this issue

Well I fixed my 403 issue, so I can make a PR if everyone wants, but I don't think it will be accepted as the maintainers probably don't want to just comment out some of the things I did. 

I can fork and make a PR on the official branch anyways though, sec.. I'm pretty sure I just made some changes to src/utils.py.

**Comment by Nackophilz** [comment]
> > I agree with that, it would benefit everyone if you could make a pull request from this issue
> 
> Well I fixed my 403 issue, so I can make a PR if everyone wants, but I don't think it will be accepted as the maintainers probably don't want to just comment out some of the things I did.
> 
> I can fork and make a PR on the official branch anyways though, sec.. I'm pretty sure I just made some changes to src/utils.py.



You can always try to make a pull request, it doesn't cost anything to try, anyway, as you said, it's always a cat and mouse game to add a workaround to cloudflare's limitations, so I think whatever the solution is if it's functional, the repo owner should welcome it ^^

**Comment by digitalnomad91** [comment]
Done: #842

**Comment by MrAlucardDante** [comment]
> Done: #842

Unfortunately, the fix in your PR isn't working, at least in docker.

I manually modified both files inside the container and I still get the same issue.

Hopefully, the fix from undetected_chromedriver will fix this

**Comment by digitalnomad91** [comment]
> > Done: #842
> 
> Unfortunately, the fix in your PR isn't working, at least in docker.
> 
> I manually modified both files inside the container and I still get the same issue.
> 
> Hopefully, the fix from undetected_chromedriver will fix this
> 
> 

Most likely related to how I modified the path for chromium option, maybe try changing that back to the helper function that I commented out? What exactly are you seeing occur? Any errors or anything that stands out? Or is it just the same redirect and timeout issue from before? 

Also what command are you using to build the image/ container btw? I have it running directly as a service and it's working well, but it could also just be run in detached mode or a tmux session for now, as this is likely to be a temporary workaround anyhow.

**Comment by MrAlucardDante** [comment]
> > > Done: #842
> > 
> > 
> > Unfortunately, the fix in your PR isn't working, at least in docker.
> > I manually modified both files inside the container and I still get the same issue.
> > Hopefully, the fix from undetected_chromedriver will fix this
> 
> Most likely related to how I modified the path for chromium option, maybe try changing that back to the helper function that I commented out? What exactly are you seeing occur? Any errors or anything that stands out? Or is it just the same redirect and timeout issue from before?
> 
> Also what command are you using to build the image/ container btw? I have it running directly as a service and it's working well, but it could also just be run in detached mode or a tmux session for now, as this is likely to be a temporary workaround anyhow.

I haven’t built a new image. I just copied the file from the container to my machine using docker cp, modified the content then pushed the file back inside the container and restarted it.

I don’t think it’s due to the chromium path option since the chromium version appears in the log. I still have the message that the captcha could be found apter multiple attempts. 

Fortunately, the torrent website I call with flaresolverr can be configured with a cookie, which bypasses cloudlflare entirely, so that fixed my issue temporarily, until the cookie expires

**Comment by thebetauser** [comment]
I also had brief success changing the options inside utils.py however after a few hours CF started flagging FS again

**Comment by MohamadBayz** [comment]
Same issue here, no workaround since i only has access to the terminal in my server

**Comment by damien2405** [comment]
Tried this solution, with docker, don't work

**Comment by digitalnomad91** [comment]
To everyone running it within docker, are you copying utils.py + flaresolverr.py? There change that made it work is one line in flaresolverr.py, but I had to comment out and change the chromium path for it to work. The chromium path change is probably more related to my machine. Have you guys tried testing it out by running python3 src/flaresolverr.py? Or just the single Python example in the other repo I linked? I need to check if it's still working or not, but if CF already defeated this then that's quite unfortunate. If true then I guess we will have to wait for the other solution that is being worked on, or setup something like the webextension that I shared earlier (which isn't the cleaned solution, but it works).

**Comment by tupac55** [comment]
> To everyone running it within docker, are you copying utils.py + flaresolverr.py? There change that made it work is one line in flaresolverr.py, but I had to comment out and change the chromium path for it to work. The chromium path change is probably more related to my machine. Have you guys tried testing it out by running python3 src/flaresolverr.py? Or just the single Python example in the other repo I linked? I need to check if it's still working or not, but if CF already defeated this then that's quite unfortunate. If true then I guess we will have to wait for the other solution that is being worked on, or setup something like the webextension that I shared earlier (which isn't the cleaned solution, but it works).

I can't change the file within docker in my Synology nas.. So can you build a docker image so i can import and test it ?

**Comment by MrAlucardDante** [comment]
> To everyone running it within docker, are you copying utils.py + flaresolverr.py? There change that made it work is one line in flaresolverr.py, but I had to comment out and change the chromium path for it to work. The chromium path change is probably more related to my machine. Have you guys tried testing it out by running python3 src/flaresolverr.py? Or just the single Python example in the other repo I linked? I need to check if it's still working or not, but if CF already defeated this then that's quite unfortunate. If true then I guess we will have to wait for the other solution that is being worked on, or setup something like the webextension that I shared earlier (which isn't the cleaned solution, but it works).

I will try to change only the service tomorrow and let you know.

**Comment by digitalnomad91** [comment]
I just tested and it's stll working for me on my hetzner machines and @ home. If you are triggering an actual captcha then I don't believe flaresolverr can defeat that atm. Perhaps something like this could work & be integrated in somehow:  
  
[https://addons.mozilla.org/en-US/firefox/addon/buster-captcha-solver/](https://addons.mozilla.org/en-US/firefox/addon/buster-captcha-solver/)  
  
I'm not sure if it's able to solve CF captchas, but if so then it could be useful in the webextension hacky workaround I shared as well. Fortunately for my use case I am not running into actual captchas, just the CF redirect check if you're human thing. 

@tupac55 @damien2405 @MrAlucardDante try this if you want to run through docker:

```shell
git clone https://github.com/digitalnomad91/FlareSolverr && cd FlareSolverr
git checkout cloudflare_fix_3
docker build --file Dockerfile -t digitalnomad91/flaresolverr:cloudflare_fix_3 .
docker rm -f flaresolverr
docker run -d   --name=flaresolverr  -p 8191:8191  -e LOG_LEVEL=info  --restart unless-stopped  digitalnomad91/flaresolverr:cloudflare_fix_3
docker logs flaresolverr -f
```

**Comment by damien2405** [comment]
> 

For log ``` Traceback (most recent call last):
  File "/app/flaresolverr.py", line 9, in <module>
    from bottle_plugins.error_plugin import error_plugin
ModuleNotFoundError: No module named 'bottle_plugins.error_plugin'
Traceback (most recent call last):
  File "/app/flaresolverr.py", line 9, in <module>
    from bottle_plugins.error_plugin import error_plugin
ModuleNotFoundError: No module named 'bottle_plugins.error_plugin'
Traceback (most recent call last):
  File "/app/flaresolverr.py", line 9, in <module>
    from bottle_plugins.error_plugin import error_plugin
ModuleNotFoundError: No module named 'bottle_plugins.error_plugin'
Traceback (most recent call last):
  File "/app/flaresolverr.py", line 9, in <module>
    from bottle_plugins.error_plugin import error_plugin
ModuleNotFoundError: No module named 'bottle_plugins.error_plugin'
Traceback (most recent call last):
  File "/app/flaresolverr.py", line 9, in <module>
    from bottle_plugins.error_plugin import error_plugin
ModuleNotFoundError: No module named 'bottle_plugins.error_plugin'
Traceback (most recent call last):
  File "/app/flaresolverr.py", line 9, in <module>
    from bottle_plugins.error_plugin import error_plugin
ModuleNotFoundError: No module named 'bottle_plugins.error_plugin'
Traceback (most recent call last):
  File "/app/flaresolverr.py", line 9, in <module>
    from bottle_plugins.error_plugin import error_plugin
ModuleNotFoundError: No module named 'bottle_plugins.error_plugin'
Traceback (most recent call last):
  File "/app/flaresolverr.py", line 9, in <module>
    from bottle_plugins.error_plugin import error_plugin
ModuleNotFoundError: No module named 'bottle_plugins.error_plugin'
Traceback (most recent call last):
  File "/app/flaresolverr.py", line 9, in <module>
    from bottle_plugins.error_plugin import error_plugin
ModuleNotFoundError: No module named 'bottle_plugins.error_plugin'
Traceback (most recent call last):
  File "/app/flaresolverr.py", line 9, in <module>
    from bottle_plugins.error_plugin import error_plugin
ModuleNotFoundError: No module named 'bottle_plugins.error_plugin' ```

**Comment by damien2405** [comment]
and idk why, 
![image](https://github.com/FlareSolverr/FlareSolverr/assets/88164999/ed21d2f6-f6d5-4915-9a17-f328c4ee7555)

No ip, so can't connect

**Comment by antivirtel** [comment]
Did you install the Python requirements? In the container, at the root of the source code: `pip install -U -r requirements.txt` - of course if you have pyenv/venv in the container use that.

**Comment by damien2405** [comment]
Sorry i'm really new with docker, i've just do this steps : 
`git clone https://github.com/digitalnomad91/FlareSolverr && cd FlareSolverr
git checkout cloudflare_fix_3
docker build --file Dockerfile -t digitalnomad91/flaresolverr:cloudflare_fix_3 .
docker rm -f flaresolverr
docker run -d   --name=flaresolverr  -p 8191:8191  -e LOG_LEVEL=info  --restart unless-stopped  digitalnomad91/flaresolverr:cloudflare_fix_3
docker logs flaresolverr -f`

Need to do something else ? And if yes, where, and how ? ^^ Sorry

**Comment by antivirtel** [comment]
The Dockerfile has it, when you build the image you run those:
> - docker build --file Dockerfile -t digitalnomad91/flaresolverr:cloudflare_fix_3 .

https://github.com/digitalnomad91/FlareSolverr/blob/cloudflare_fix_3/Dockerfile#L45

I'm not using Docker and It actually goes forward for me after installing stuff from the [requirements.txt](https://github.com/digitalnomad91/FlareSolverr/blob/cloudflare_fix_3/test-requirements.txt), but maybe you need the [test-requirements.txt](https://github.com/digitalnomad91/FlareSolverr/blob/cloudflare_fix_3/test-requirements.txt) too?  Just guessing here. 

Might need to run that manually in the container, or just add another RUN line to the Dockerfile right after the other:

```
pip install -U -r test-requirements.txt
```

**Comment by mushuthecat** [comment]
Hello,

im using docker and try this workaround:

download this resources: https://github.com/digitalnomad91/FlareSolverr-udetect-patch/
Then, docker cp thos two files /src/utils.py and /src/flaresolverr.py to my docker container in /tmp
Then, i performed a mv from my old files flaresolverr.py and utils.py with .bak extension.
Then i performed a cp from the new files previously pushed into the /tmp docker container to the same place as the other files previously move to .bak
After that i run a pip install -U -R /tmp/requirements.txt previously docker cp to my /tmp container

To finish i exit the docker exec and performed a docker-compose restart of my running container.

This is not working still have the same issue  Error: Error solving the challenge. Timeout after 90.0 seconds.
2023-08-02 16:02:09 INFO     Response in 90.911 s


Did i miss something ?

Thanks

**Comment by thebetauser** [comment]
I got it working using the steps from https://github.com/FlareSolverr/FlareSolverr/issues/811#issuecomment-1661369698 however you need to update the requirements.txt file to get it working.

### Old
>selenium==4.9.1
### New
>selenium

Remove the "==4.9.1" line and rebuild the docker container

>2023-08-02T16:59:09.612424725Z 2023-08-02 16:59:09 INFO     ReqId 140586149472000 Challenge detected. Title found: Just a moment...
>2023-08-02T16:59:09.612873035Z 2023-08-02 16:59:09 DEBUG    ReqId 140586149472000 Waiting for title (attempt 1): Just a moment...
>2023-08-02T16:59:19.901362094Z 2023-08-02 16:59:19 DEBUG    ReqId 140586149472000 Timeout waiting for selector
>2023-08-02T16:59:19.902048814Z 2023-08-02 16:59:19 DEBUG    ReqId 140586149472000 Try to find the Cloudflare verify checkbox
>2023-08-02T16:59:19.984131935Z 2023-08-02 16:59:19 DEBUG    ReqId 140586149472000 Cloudflare verify checkbox not found on the page
>2023-08-02T16:59:19.988786694Z 2023-08-02 16:59:19 DEBUG    ReqId 140586149472000 Try to find the Cloudflare 'Verify you are human' button
>2023-08-02T16:59:20.015487658Z 2023-08-02 16:59:20 DEBUG    ReqId 140586149472000 The Cloudflare 'Verify you are human' button not found on the page
>2023-08-02T16:59:22.033806081Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for title (attempt 2): Just a moment...
>2023-08-02T16:59:22.043024029Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for title (attempt 2): DDoS-Guard
>2023-08-02T16:59:22.049956928Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for selector (attempt 2): #cf-challenge-running
>2023-08-02T16:59:22.069154293Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for selector (attempt 2): .ray_id
>2023-08-02T16:59:22.089596029Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for selector (attempt 2): .attack-box
>2023-08-02T16:59:22.108753064Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for selector (attempt 2): #cf-please-wait
>2023-08-02T16:59:22.127494530Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for selector (attempt 2): #challenge-spinner
>2023-08-02T16:59:22.145520295Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for selector (attempt 2): #trk_jschal_js
>2023-08-02T16:59:22.166125521Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for selector (attempt 2): td.info #js_info
>2023-08-02T16:59:22.188443346Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for selector (attempt 2): div.vc div.text-box h2
>2023-08-02T16:59:22.207338232Z 2023-08-02 16:59:22 DEBUG    ReqId 140586149472000 Waiting for redirect
>2023-08-02T16:59:32.475915675Z 2023-08-02 16:59:32 DEBUG    ReqId 140586149472000 Timeout waiting for redirect
>2023-08-02T16:59:32.476575045Z 2023-08-02 16:59:32 INFO     ReqId 140586149472000 Challenge solved!

**Comment by Raiiyzen** [comment]
> 

still no working with https://www3.yggtorrent.wtf/ 
with prowlarr: 

> 2023-08-02 20:45:19 INFO     Challenge not detected!
2023-08-02 20:45:19 INFO     Response in 6.725 s
2023-08-02 20:45:19 INFO     192.168.10.247 POST http://192.168.10.21:8191/v1 200 OK

log from prowlarr: 

> Unable to connect to indexer, check the log above the ValidationFailure for more details. Login failed: No form found on https://www3.yggtorrent.wtf/ using form selector #user-login

**Comment by damien2405** [comment]
> > 
> 
> still no working with https://www3.yggtorrent.wtf/ with prowlarr:
> 
> > 2023-08-02 20:45:19 INFO     Challenge not detected!
> > 2023-08-02 20:45:19 INFO     Response in 6.725 s
> > 2023-08-02 20:45:19 INFO     192.168.10.247 POST http://flaresolverr:8191/v1 200 OK
> 
> log from prowlarr:
> 
> > Unable to connect to indexer, check the log above the ValidationFailure for more details. Login failed: No form found on https://www3.yggtorrent.wtf/ using form selector #user-login


That wierd, everything's fine, with a lot of people on YGG

**Comment by Raiiyzen** [comment]
> > > 
> > 
> > 
> > still no working with https://www3.yggtorrent.wtf/ with prowlarr:
> > > 2023-08-02 20:45:19 INFO     Challenge not detected!
> > > 2023-08-02 20:45:19 INFO     Response in 6.725 s
> > > 2023-08-02 20:45:19 INFO     192.168.10.247 POST http://flaresolverr:8191/v1 200 OK
> > 
> > 
> > log from prowlarr:
> > > Unable to connect to indexer, check the log above the ValidationFailure for more details. Login failed: No form found on https://www3.yggtorrent.wtf/ using form selector #user-login
> 
> That wierd, everything's fine, with a lot of people on YGG

hmmmm, are you sure ? with the recent activation of cloudflare for this website few days ago, i think i'm not the only one

**Comment by damien2405** [comment]
> > > > 
> > > 
> > > 
> > > still no working with https://www3.yggtorrent.wtf/ with prowlarr:
> > > > 2023-08-02 20:45:19 INFO     Challenge not detected!
> > > > 2023-08-02 20:45:19 INFO     Response in 6.725 s
> > > > 2023-08-02 20:45:19 INFO     192.168.10.247 POST http://flaresolverr:8191/v1 200 OK
> > > 
> > > 
> > > log from prowlarr:
> > > > Unable to connect to indexer, check the log above the ValidationFailure for more details. Login failed: No form found on https://www3.yggtorrent.wtf/ using form selector #user-login
> > 
> > 
> > That wierd, everything's fine, with a lot of people on YGG
> 
> hmmmm, are you sure ? with the recent activation of cloudflare for this website few days ago, i think i'm not the only one

GO check on the forum, everything is okay with a lot of people, with the new uptate, since this afternoon

**Comment by Raiiyzen** [comment]
> Even with 2023-08-02 18:00:09 INFO Challenge not detected! its working, try with jackett, prowlarr is really shitty


with jackett:

> An error occurred while testing this indexer
Exception (yggtorrent): The cookies provided by FlareSolverr are not valid: The cookies provided by FlareSolverr are not valid

**Comment by thebetauser** [comment]
The site is down which is why it's not working. I think all the users who updated to the new version are flooding the site, Give it a few hours see if it comes back

**Comment by MohamadBayz** [comment]
It still doesn't work for me with `https://torrentqq267.com/`
Log:
`2023-08-02 19:05:23 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://torrentqq267.com/torrent/newest.html', 'proxy': {}}
2023-08-02 19:05:27 INFO     Challenge detected. Title found: Just a moment...
2023-08-02 19:06:24 ERROR    Error: Error solving the challenge. Timeout after 60.0 seconds.
2023-08-02 19:06:24 INFO     Response in 60.74 s
2023-08-02 19:06:24 INFO     10.0.2.100 POST http://localhost:12499/v1 500 Internal Server Error`

**Comment by MohamadBayz** [comment]
Full log with debug level
[Logs.txt](https://github.com/FlareSolverr/FlareSolverr/files/12243843/Logs.txt)

**Comment by MohamadBayz** [comment]
Can't find element selenium issue

**Comment by MohamadBayz** [comment]
Sorry here its forgot to save:
[Logs.txt](https://github.com/FlareSolverr/FlareSolverr/files/12243864/Logs.txt)

**Comment by Raiiyzen** [comment]
> The site is down which is why it's not working. I think all the users who updated to the new version are flooding the site, Give it a few hours see if it comes back

ok now it's working well with the latest version, thanks !

**Comment by digitalnomad91** [comment]
@Raiiyzen When I try to load the site like normal at home I get a "You've been blocked" page by CloudFlare. Da fuq.....

**Comment by ilike2burnthing** [comment]
@MohamadBayz can you try iDope, LePorno.info, and Torrent[CORE], see if they all fail? If not, try TorrentQQ again after, see if it still fails.

Are you running from source or using the pre-compiled binaries?

**Comment by ilike2burnthing** [comment]
I managed to get the same/similar error on Windows, but on the first retry it then worked fine:

```text
C:\Users\WDAGUtilityAccount\Downloads\FlareSolverr-master>python src/flaresolverr.py
2023-08-02 22:49:52 INFO     ReqId 4448 FlareSolverr 3.3.0
2023-08-02 22:49:52 DEBUG    ReqId 4448 Debug log enabled
2023-08-02 22:49:52 INFO     ReqId 4448 Testing web browser installation...
2023-08-02 22:49:52 INFO     ReqId 4448 Platform: Windows-10-10.0.19041-SP0
2023-08-02 22:49:52 INFO     ReqId 4448 Chrome / Chromium path: C:\Program Files\Google\Chrome\Application\chrome.exe
2023-08-02 22:49:52 INFO     ReqId 4448 Chrome / Chromium major version: 115
2023-08-02 22:49:52 INFO     ReqId 4448 Launching web browser...
2023-08-02 22:49:52 DEBUG    ReqId 4448 Launching web browser...
2023-08-02 22:49:52 DEBUG    ReqId 4448 Started executable: `C:\Users\WDAGUtilityAccount\appdata\roaming\undetected_chromedriver\chromedriver.exe` in a child process with pid: 6312
2023-08-02 22:49:53 INFO     ReqId 4448 FlareSolverr User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36
2023-08-02 22:49:53 INFO     ReqId 4448 Test successful!
2023-08-02 22:49:53 INFO     ReqId 4448 Serving on http://0.0.0.0:8191
2023-08-02 22:50:05 INFO     ReqId 4920 Incoming request => POST /v1 body: {'maxTimeout': 55000, 'cmd': 'request.get', 'url': 'https://helltorrents.com/login.php'}
2023-08-02 22:50:05 DEBUG    ReqId 4920 Launching web browser...
2023-08-02 22:50:05 DEBUG    ReqId 4920 Started executable: `C:\Users\WDAGUtilityAccount\appdata\roaming\undetected_chromedriver\chromedriver.exe` in a child process with pid: 2752
2023-08-02 22:50:05 DEBUG    ReqId 4920 New instance of webdriver has been created to perform the request
2023-08-02 22:50:05 DEBUG    ReqId 7300 Navigating to... https://helltorrents.com/login.php
2023-08-02 22:50:12 DEBUG    ReqId 7300 Started executable: `C:\Users\WDAGUtilityAccount\appdata\roaming\undetected_chromedriver\chromedriver.exe` in a child process with pid: 6820
2023-08-02 22:50:12 INFO     ReqId 7300 Challenge detected. Title found: Just a moment...
2023-08-02 22:50:12 DEBUG    ReqId 7300 Waiting for title (attempt 1): Just a moment...
2023-08-02 22:50:23 DEBUG    ReqId 7300 Timeout waiting for selector
2023-08-02 22:50:23 DEBUG    ReqId 7300 Try to find the Cloudflare verify checkbox...
2023-08-02 22:50:23 DEBUG    ReqId 7300 Cloudflare verify checkbox found and clicked!
2023-08-02 22:50:23 DEBUG    ReqId 7300 Try to find the Cloudflare 'Verify you are human' button...
2023-08-02 22:50:23 DEBUG    ReqId 7300 The Cloudflare 'Verify you are human' button not found on the page. Error: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//input[@type='button' and @value='Verify you are human']"}
  (Session info: chrome=115.0.5790.171); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
Backtrace:
        GetHandleVerifier [0x009FA813+48355]
        (No symbol) [0x0098C4B1]
        (No symbol) [0x00895358]
        (No symbol) [0x008C09A5]
        (No symbol) [0x008C0B3B]
        (No symbol) [0x008EE232]
        (No symbol) [0x008DA784]
        (No symbol) [0x008EC922]
        (No symbol) [0x008DA536]
        (No symbol) [0x008B82DC]
        (No symbol) [0x008B93DD]
        GetHandleVerifier [0x00C5AABD+2539405]
        GetHandleVerifier [0x00C9A78F+2800735]
        GetHandleVerifier [0x00C9456C+2775612]
        GetHandleVerifier [0x00A851E0+616112]
        (No symbol) [0x00995F8C]
        (No symbol) [0x00992328]
        (No symbol) [0x0099240B]
        (No symbol) [0x00984FF7]
        BaseThreadInitThunk [0x75C200C9+25]
        RtlGetAppContainerNamedObjectPath [0x77987B1E+286]
        RtlGetAppContainerNamedObjectPath [0x77987AEE+238]

2023-08-02 22:50:26 DEBUG    ReqId 7300 Waiting for title (attempt 2): Just a moment...
2023-08-02 22:50:26 DEBUG    ReqId 7300 Waiting for title (attempt 2): DDoS-Guard
2023-08-02 22:50:26 DEBUG    ReqId 7300 Waiting for selector (attempt 2): #cf-challenge-running
2023-08-02 22:50:26 DEBUG    ReqId 7300 Waiting for selector (attempt 2): .ray_id
2023-08-02 22:50:27 DEBUG    ReqId 7300 Waiting for selector (attempt 2): .attack-box
2023-08-02 22:50:27 DEBUG    ReqId 7300 Waiting for selector (attempt 2): #cf-please-wait
2023-08-02 22:50:27 DEBUG    ReqId 7300 Waiting for selector (attempt 2): #challenge-spinner
2023-08-02 22:50:27 DEBUG    ReqId 7300 Waiting for selector (attempt 2): #trk_jschal_js
2023-08-02 22:50:27 DEBUG    ReqId 7300 Waiting for selector (attempt 2): td.info #js_info
2023-08-02 22:50:27 DEBUG    ReqId 7300 Waiting for selector (attempt 2): div.vc div.text-box h2
2023-08-02 22:50:27 DEBUG    ReqId 7300 Waiting for redirect
2023-08-02 22:50:37 DEBUG    ReqId 7300 Timeout waiting for redirect
2023-08-02 22:50:37 INFO     ReqId 7300 Challenge solved!
2023-08-02 22:50:37 DEBUG    ReqId 4920 A used instance of webdriver has been destroyed
2023-08-02 22:50:37 DEBUG    ReqId 4920 Response => POST /v1 body: {'status': 'ok', 'message': 'Challenge solved!', 'solution': {'url': 'https://helltorrents.com/login.php', 'status': 200, 'cookies': [{'domain': 'helltorrents.com', 'expiry': 1696197025, 'httpOnly': False, 'name': 'adcashufpv3', 'path': '/', 'sameSite': 'Lax', 'secure': False, 'value': '12517460661122637838370731707'}, {'domain': '.helltorrents.com', 'expiry': 1722549024, 'httpOnly': True, 'name': 'cf_clearance', 'path': '/', 'sameSite': 'None', 'secure': True, 'value': 'nF8M3SGdRNPOw7pPTrl4w9aYYUEbjBVckb1cfkTz1zk-1691013007-0-1-dd2220e2.429c2e87.8a237f02-250.0.0'}], 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36', 'headers': {}, 'response': '[...]'}, 'startTimestamp': 1691013005261, 'endTimestamp': 1691013037918, 'version': '3.3.0'}
2023-08-02 22:50:37 INFO     ReqId 4920 Response in 32.657 s
2023-08-02 22:50:37 INFO     ReqId 4920 127.0.0.1 POST http://127.0.0.1:8191/v1 200 OK
```

**Comment by ilike2burnthing** [comment]
@ngosang ^

**Comment by MohamadBayz** [comment]
> @MohamadBayz can you try iDope, LePorno.info, and Torrent[CORE], see if they all fail? If not, try TorrentQQ again after, see if it still fails.
> 
Tried all of them they don't work, and Torrentqq still fails
Here's the log
[myFile.log](https://github.com/FlareSolverr/FlareSolverr/files/12245329/myFile.log)

> Are you running from source or using the pre-compiled binaries?

I think it's a pre compiled i use a docker container and I setup flaresolverr using:
` services:
     flaresolverr:
         image: ghcr.io/flaresolverr/flaresolverr:latest
         environment:
           - LOG_LEVEL=${LOG_LEVEL:-debug}
           - LOG_HTML=${LOG_HTML:-false}
           - CAPTCHA_SOLVER=${CAPTCHA_SOLVER:-none}
           - TZ=UTC
         ports:
           - 127.0.0.1:12499:8191
         restart: unless-stopped
`

**Comment by thebetauser** [comment]
I tested this on idope and it worked
>2023-08-03T01:03:19.158512344Z 2023-08-03 01:03:19 INFO     Challenge detected. Title found: Just a moment...
>2023-08-03T01:03:44.401121172Z 2023-08-03 01:03:44 INFO     Challenge solved!
>2023-08-03T01:03:44.423326597Z 2023-08-03 01:03:44 INFO     Response in 41.106 s

same with torrentqq
>2023-08-03T01:04:53.943064106Z 2023-08-03 01:04:53 INFO     Challenge detected. Title found: Just a moment...
>2023-08-03T01:04:57.752332223Z 2023-08-03 01:04:57 INFO     Challenge solved!
>2023-08-03T01:04:57.806408910Z 2023-08-03 01:04:57 INFO     Response in 8.135 s

**Comment by MohamadBayz** [comment]
> 
> Edit: Tested on v3.3.0 docker image running ubuntu

How did you update from 3.2 to 3.3? Did you do anything special or reinstalled the requirements.txt?

**Comment by mushuthecat** [comment]
Hi,

Please check your docker-compose.yaml

This is an extract of my version make sure to use the latest version also...

version: '3'

services:
  flaresolverr:
    image: flaresolverr/flaresolverr:latest
    container_name: flaresolverr
    environment:
      - CAPTCHA_SOLVER=hcaptcha-solver
    dns:
      - 1.1.1.1
      - 1.0.0.1
    restart: always

Le jeu. 3 août 2023 à 17:08, MohamadBayz ***@***.***> a
écrit :

**Comment by MohamadBayz** [comment]
Will try thanks

**Comment by Zegorax** [comment]
I'm on 3.3.2 and I have the same issue

**Comment by khooniatma** [comment]
I am on 3.3.4 on Windows and seeing the same issue. 

```
2023-09-16 11:38:50 INFO     ReqId 13240 FlareSolverr 3.3.4
2023-09-16 11:38:50 DEBUG    ReqId 13240 Debug log enabled
2023-09-16 11:38:50 INFO     ReqId 13240 Testing web browser installation...
2023-09-16 11:38:50 INFO     ReqId 13240 Platform: Windows-10-10.0.22621-SP0
2023-09-16 11:38:50 INFO     ReqId 13240 Chrome / Chromium path: C:\Users\Administrator\Documents\flaresolverr_windows_x64\flaresolverr\chrome\chrome.exe
2023-09-16 11:38:50 INFO     ReqId 13240 Chrome / Chromium major version: 115
2023-09-16 11:38:50 INFO     ReqId 13240 Launching web browser...
2023-09-16 11:38:50 DEBUG    ReqId 13240 Launching web browser...
2023-09-16 11:38:54 DEBUG    ReqId 13240 Started executable: `C:\Users\Administrator\appdata\roaming\undetected_chromedriver\undetected_chromedriver.exe` in a child process with pid: 16936
2023-09-16 11:38:54 INFO     ReqId 13240 FlareSolverr User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36
2023-09-16 11:38:54 INFO     ReqId 13240 Test successful!
2023-09-16 11:38:54 INFO     ReqId 13240 Serving on http://0.0.0.0:8191
2023-09-16 11:42:01 INFO     ReqId 6708 Incoming request => POST /v1 body: {'maxTimeout': 150000, 'cmd': 'request.get', 'url': 'https://bitsearch.nocensor.cloud/search?sort=date&order=desc&limit=100'}
2023-09-16 11:42:01 DEBUG    ReqId 6708 Launching web browser...
2023-09-16 11:42:01 DEBUG    ReqId 6708 Started executable: `C:\Users\Administrator\appdata\roaming\undetected_chromedriver\chromedriver.exe` in a child process with pid: 18352
2023-09-16 11:42:02 DEBUG    ReqId 6708 New instance of webdriver has been created to perform the request
2023-09-16 11:42:02 DEBUG    ReqId 12592 Navigating to... https://bitsearch.nocensor.cloud/search?sort=date&order=desc&limit=100
2023-09-16 11:42:03 DEBUG    ReqId 12592 Response HTML:
<html lang="en-US" class="lang-en"><head><title>Just a moment...</title><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><meta name="robots" content="noindex,nofollow"><meta name="viewport" content="width=device-width,initial-scale=1"><link href="/cdn-cgi/styles/challenges.css" rel="stylesheet"><script src="/cdn-cgi/challenge-platform/h/b/orchestrate/chl_page/v1?ray=8076f49faed318fd"></script><script src="https://challenges.cloudflare.com/turnstile/v0/b/8370c0b3/api.js?onload=wcgW6&amp;render=explicit" async="" defer="" crossorigin="anonymous"></script></head><body class="no-js"><div class="main-wrapper" role="main"><div class="main-content"><h1 class="zone-name-title h1">bitsearch.nocensor.cloud</h1><h2 id="challenge-running" class="h2">Checking if the site connection is secure</h2><div id="challenge-stage" style="display: block;"><div id="turnstile-wrapper" class="captcha-prompt spacer"><div><iframe src="https://challenges.cloudflare.com/cdn-cgi/challenge-platform/h/b/turnstile/if/ov2/av0/rcv0/0/satfl/0x4AAAAAAADnPIDROrmt1Wwj/light/normal" allow="cross-origin-isolated; fullscreen" sandbox="allow-same-origin allow-scripts allow-popups" id="cf-chl-widget-satfl" tabindex="0" title="Widget containing a Cloudflare security challenge" style="border: none; overflow: hidden; width: 300px; height: 65px;"></iframe><input type="hidden" name="cf-turnstile-response" id="cf-chl-widget-satfl_response"></div></div></div><div id="challenge-spinner" class="spacer loading-spinner" style="display: none; visibility: hidden;"><div class="lds-ring"><div></div><div></div><div></div><div></div></div></div><div id="challenge-body-text" class="core-msg spacer">bitsearch.nocensor.cloud needs to review the security of your connection before proceeding.</div><div id="challenge-success" style="display: none;"><div class="h2"><span class="icon-wrapper"><img class="heading-icon" alt="Success icon" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADQAAAA0CAMAAADypuvZAAAANlBMVEUAAAAxMTEwMDAxMTExMTEwMDAwMDAwMDAxMTExMTExMTEwMDAwMDAxMTExMTEwMDAwMDAxMTHB9N+uAAAAEXRSTlMA3zDvfyBAEJC/n3BQz69gX7VMkcMAAAGySURBVEjHnZZbFoMgDEQJiDzVuv/NtgbtFGuQ4/zUKpeMIQbUhXSKE5l1XSn4pFWHRm/WShT1HRLWC01LGxFEVkCc30eYkLJ1Sjk9pvkw690VY6k8DWP9OM9yMG0Koi+mi8XA36NXmW0UXra4eJ3iwHfrfXVlgL0NqqGBHdqfeQhMmyJ48WDuKP81h3+SMPeRKkJcSXiLUK4XTHCjESOnz1VUXQoc6lgi2x4cI5aTQ201Mt8wHysI5fc05M5c81uZEtHcMKhxZ7iYEty1GfhLvGKpm+EYkdGxm1F5axmcB93DoORIbXfdN7f+hlFuyxtDP+sxtBnF43cIYwaZAWRgzxIoiXEMESoPlMhwLRDXeK772CAzXEdBRV7cmnoVBp0OSlyGidEzJTFq5hhcsA5388oSGM6b5p+qjpZrBlMS9xj4AwXmz108ukU1IomM3ceiW0CDwHCqp1NjAqXlFrbga+xuloQJ+tuyfbIBPNpqnmxqT7dPaOnZqBfhSBCteJAxWj58zLk2xgg+SPGYM6dRO6WczSnIxxwEExRaO+UyCUhbOp7CGQ+kxSUfNtLQFC+Po29vvy7jj4y0yAAAAABJRU5ErkJggg=="></span>Connection is secure</div><div class="core-msg spacer">Proceeding...</div></div><noscript><div id="challenge-error-title"><div class="h2"><span class="icon-wrapper"><div class="heading-icon warning-icon"></div></span><span id="challenge-error-text">Enable JavaScript and cookies to continue</span></div></div></noscript></div></div><script>(function(){window._cf_chl_opt={cvId: '2',cZone: "bitsearch.nocensor.cloud",cType: 'managed',cNounce: '2460',cRay: '8076f49faed318fd',cHash: 'dcd88581265a125',cUPMDTk: "\/search?sort=date&order=desc&limit=100&__cf_chl_tk=3vCA1M5s9e9t2DHqy9coLNvDY6LhSLtlk0S8ZpZb0Jc-1694844723-0-gaNycGzNDVA",cFPWv: 'b',cTTimeMs: '1000',cMTimeMs: '0',cTplV: 5,cTplB: 'cf',cK: "visitor-time",fa: "\/search?sort=date&order=desc&limit=100&__cf_chl_f_tk=3vCA1M5s9e9t2DHqy9coLNvDY6LhSLtlk0S8ZpZb0Jc-1694844723-0-gaNycGzNDVA",md: "_Psrz2MzvBcL4nHsysihQHt3JJ7HOnWutgboHcSzLv0-1694844723-0-AZ05VIi-rWBjI7auwfsThYA9Wwv8MwDaFUtKwfW5ssr-PJEGIC73kgl1Z6tUoW2fad8_iPQFlkGyum4xiW7bQg0glE0ZJYD577otiUHXHYkan8AHiD_maZTU9KeWU3hM-nXQL2pLKwkbFXhJKn0M5LjAPREPlv_SyXQOk7VK7bnbgveHl-hBg0KJ21m9VgPl6kU8OMAnVSitldQnmA9oPGmhmPfo2OWQnOu1nl92B7Wlo2IrjrMDitN4rD0YGTcZTbcq-7SZ8qSZybO0tqooPTlS8jo24oEhrftV-L6MlaU8Dqnc7wgn6UHdj5eGYduqvkwQhN9UlmMenPmoGqXHP7Z317AJGzjHaK2dnvmQWdh2OLKXFgQQME0Ym3m6pwFoSmyLUDOCsnbpOTub9ylX1Wh-i-ik6L6UD3hSlrFpffAWGPZTzpC6BhxWs2Zabj_WVBsq1SpfUQrM94Imlscwaen5xx-kPoi6Jv4-GrY02MmlUr6r5JiTPkdVGfZWUx1YPk0TPGlVaiPD9qlRVdf1y39McuHMzcziH9bMwNHk3SkaK_WjmMFvxsvtzAcqlnVUdfi1SNigwZs5Mu0RXgWpxaUDNIS9suNGVvE2V8y0ONSiaJvGZEykLJfo0EsckCc-f-DX6DbOQ3ranL6KPTmxwt8eCw_CHRTBY6ft9CUKPhjCPrFn9721xgG99schKwzjNSAXDuQkwVKpTEqj1qB6v7A1AIKwiqP-nQO5rlolqWdB1eyiuMoQjMZyoDzsNe2QwmF7j88sz9_eMzsXOxgpWSry2D10e0FJBMEnajz_LY6f4f4n-wCPFH9Jz_jdnroI6pUUeWiUZ7fqz5sk2gZsD5NGRCQPpcMaML4inU3u_SJqCSOZZCNlHk3aRP7MqA-5r--OZy6FMj1GqIhmrSIYXzR40tdWozs6DJA_S5rNPH3c7kb9x7kYt3bGMhlZ1S31F7vIH9oU7SzDqxx7bLk2zqb3Gg1r2Ip3J4CurOd0CKgmDPC5V5vtbBTAyG8C7Yl-Azl6-9nBBE_2dHp2dxDS3CS6zvahcKneYvgOT8V2-olRgYv7y0CmprjiIyL-wLgXJbEDMk6PvN5QqCcxd95DXIdNcJ_tPHF1vzAav81meZ4zRGYnPffC_7vfYgOSO13FS1stanwpxJ-QAQCHBhOyeVR8AEgRIunvUMSGmMErA2BCIPvzQFc0g5OYnyESGrKVxYY6PauNheub8hBkf5QTJyIPTnc2coqO4OknsWPRWd9mlGJOj_JIxnB8g0T-ap7G7h6P-7hwlnzoimJJl7uKp-on7O5tvUGeCNxWulEV2mXXbO53vAtE7c4seq5EQnzidhzC4SDaWHl4rAB-ofkcGLpewj6MFpBbeUxL7lcpsWqvgM5yjVc2ZmucBiti8bkyS3Gq9lGId5huA7_CvetIViu14-95RG2ZcvSQBKZxis_gSZl6DCO0Wr7aLk5d_zyDqLmJ-DQIww7TggsID99Q2kj6r-IJQ_jdIXUe8qzRAM-fHTVpDRjjZu9KV7BFyGI1-3WNRd98eYAmYQ8GlFL0JdlFQ5eOl4s8HEsyKDN3akTZTLjV2WTC94e1nFjzUYatta1kc1Wka50wKF4EQqCb_ncvi_r7oyMgaZTTW7zerER78vaX1WbPZwNLNUQjBHwO9J_n_yyKJHzo1QFUEllRHHFDTUigVimKtFtPtOKecKXcbhjtxXqTfXcaAlGUUzmFKlW65AK3_h84gSXhT3R-J3O4_cRwE4qi4rPW_MpjESCnPqM4TIjGloiZOgcZVUJVj9Ka7sHIkKdsKgol6gqcnob27lXO9qGkUfKXbAskhdcHh0vbc8qfjnd0Sqe-Ed8ebTNs8aiqNF7Kbku9SlNDiffMWlcaFrmACTaY9PmeQ7tIv3Q-Bnl-F-eRzvxL22TVB5PvYYxovX3ddULUDn_Z5S7TnBb0YjhAvcNf0ddVwrOUE7_1coNC1pzh2K50SVzTv6OVfrVDjabJa-Ib7xqnEa00-x8sZL6IybzD4uy_Dj5uRVaZMsEtoUHw7hKKaQ8o-72JMD0oMKiRunBu_MQAcnCWi3a_TdhrtFHQQQVRgBfht5RC47PXK7_LMr4G1HAqD6XlgEo-l1Oo1qd7zNcXjCajPdGhDImEIvieeo6lzJOcn3XacCz02mvYHreryOT8jkhJwm-aaVr581ObPZZe4xOnplSxlmKuuyQtwimtE1yNtiu5MEak0aSyk8xpuODyoDiv1gy7UoYFuUNHLH-pA4Dneb6yZgCn1Om1F75G_E06JO9k_-IC6rDs1LC2lj2CH812NPM1cCHDAu5nFyz1m4fY8owo5Ti8JuXYi30kRz1Q3_6G1ut5Wz7F2M8ZyFsJCclPGRGImaH0gDI9ZoX1ZPwzLQuNXHXqeXC2z0Iwk0w6_8EzADCdIrYo__ETaY8hU4K3ovtxSiblrqtedSG_fJlbdfICmEJRj_5Icnpl6H3rlQve045BA-doaJyEJvwa9FMMrfHIMmzRPtZIEDVFbRHmr280FSRYu-rlFEHn-sNtfyHKlUV47V7hCEILiMizFTHxD5g6MiEAIPP5AAGcixK-d8iONdjcMJ2T--U8OCjYVmx0PnxbUfVDIzo9Bl76Vj1uR1GJ77-3x5_qC6h83dev_W9AYQdGAeTq8uI0FO8X8gUZqra_6Iqs8N40sCGpCJ9IHlEtjWgXN0208_Aj94q1M3orUEqbGTwQS_tSaeUuo1R0ClTQ3HtV0TPziM4x_mAgitsfLpxKrqjxNGK_4qe5ABa7ZWWKgH1Pfn8-ox6feX85FbqeWk0pFCPEB0SPi9L0V6fb8R4T659poGv6S8EQ4fm-8HulwZ9XTEMENr5wmYA-Fj9Q1RvbTl1KCbyCbNcWpkL86dQ3u_8c8R6eMd7x-YpIBTGi8as3m_Z_ToL6GTfws3AGxWq-X7WH08AyOhRd6da5SqnA_2YAeHtSc3LUHXAc_ee7vY-5EH4pSKIpP9dlzTpBXv5RX3k4WuW0yj2Q0BP7fCh9CbeLkXMw2nmw62KIP2CI2pBrDvDmy7wZlh9tRo_au0Mp4yNyjdOjd3qbXKROEIrRyil7n5380fHeROyyOhqJsolInqKxjRwpUrV8mKQOuRRaRGbd9iKHpb-0AH0VG2Fq5XSEaWZuUIjwEIP0zPOaQQPliRLRb7rGhIUjlmQxY1ZJV8Aw09T9Zc-WfTfMqiIQXrU6spRFnEk2b7Hr811tdO89HbSuWNL8ayp4w2lrWwFo9CbI_TgraxQX0X6a_SsMMBXIEoq9pinJbespjGRqfT-COHwcKuzaFK-wZn_v4H0JnUql5Gaux0b1IHzrkEl-NeidjQ93AGk",cRq: {ru: 'aHR0cHM6Ly9iaXRzZWFyY2gubm9jZW5zb3IuY2xvdWQvc2VhcmNoP3NvcnQ9ZGF0ZSZvcmRlcj1kZXNjJmxpbWl0PTEwMA==',ra: 'TW96aWxsYS81LjAgKFdpbmRvd3MgTlQgMTAuMDsgV2luNjQ7IHg2NCkgQXBwbGVXZWJLaXQvNTM3LjM2IChLSFRNTCwgbGlrZSBHZWNrbykgQ2hyb21lLzExNS4wLjAuMCBTYWZhcmkvNTM3LjM2',rm: 'R0VU',d: 'tPUsYfh4NekaRuGqFrouCIezgojkMKh9Te24yW02q69CLSkMlScNRSuwhHxyG7EqmTf3KKY4oE41HSrgxlVLXfKxWXkS3XQ4WCrhiSn/hfiQuK3o47xUo2u9OrvIP1YGE9GGO4FneXJPIRnA6l00BNXtUY54AZIPVbaOI2rWYRA1n5QnIfPVs3H7EZv4ArAJ31Etw+UUeJbQk4bEYuS88lWWm+mvRJ3nwXrFD2hAloosZ095jb+AR6lozVwU/OAXddC0PWh0t5GBf4Mowe+mm5K3A7P8JKskMVOuFqTlFOi2jyeVyoB3H/HABntDbJvYB9GO6ekfPdiSgz3c3Lug+j+q9GS2BRwgXJ7OK38rGalIifNuZs2+wA3WeT23hUy14vaLfczxaw7V+zNfhp+V/nGqzTqgFl1OQqXB6dsgCcoVzgyK8z156rF5hkHgp5Ap41k6Isu5Kjkb62tN0GHRvLiGMQY9vMot30vD2rRMgb66tkz+BJeQjbzo204rUv5p',t: 'MTY5NDg0NDcyMy4xNDIwMDA=',cT: Math.floor(Date.now() / 1000),m: 'qIikiG+eh0Ll4fIpObiGW831XQY0qyywavjg5J7SqkM=',i1: '/Dw3aPwpr3aVU1uVgN0JAg==',i2: 'Hi8Vu9nDHwSaKwfdwt8F+w==',zh: 'Ix3eDZk0I8apnSZg8sxu0wopEDB3Y+J1lgrbErRMerI=',uh: '49m45payZ+JaK1qP7bVg0G3ztg3QEguIG+n0xYKxzGA=',hh: 'TuZ4m5lntSK5i4+xlRtSZg4YIaTHaMkRfhr4FllqIWY=',}};var cpo = document.createElement('script');cpo.src = '/cdn-cgi/challenge-platform/h/b/orchestrate/chl_page/v1?ray=8076f49faed318fd';window._cf_chl_opt.cOgUHash = location.hash === '' && location.href.indexOf('#') !== -1 ? '#' : location.hash;window._cf_chl_opt.cOgUQuery = location.search === '' && location.href.slice(0, location.href.length - window._cf_chl_opt.cOgUHash.length).indexOf('?') !== -1 ? '?' : location.search;if (window.history && window.history.replaceState) {var ogU = location.pathname + window._cf_chl_opt.cOgUQuery + window._cf_chl_opt.cOgUHash;history.replaceState(null, null, "\/search?sort=date&order=desc&limit=100&__cf_chl_rt_tk=3vCA1M5s9e9t2DHqy9coLNvDY6LhSLtlk0S8ZpZb0Jc-1694844723-0-gaNycGzNDVA" + window._cf_chl_opt.cOgUHash);cpo.onload = function() {history.replaceState(null, null, ogU);}}document.getElementsByTagName('head')[0].appendChild(cpo);}());</script><div class="footer" role="contentinfo"><div class="footer-inner"><div class="clearfix diagnostic-wrapper"><div class="ray-id">Ray ID: <code>8076f49faed318fd</code></div></div><div class="text-center" id="footer-text">Performance &amp; security by <a rel="noopener noreferrer" href="https://www.cloudflare.com?utm_source=challenge&amp;utm_campaign=m" target="_blank">Cloudflare</a></div></div></div></body></html>
2023-09-16 11:42:03 INFO     ReqId 12592 Challenge detected. Title found: Just a moment...
2023-09-16 11:42:03 DEBUG    ReqId 12592 Waiting for title (attempt 1): Just a moment...
2023-09-16 11:42:04 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:04 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:04 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:04 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:04 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:06 DEBUG    ReqId 12592 Waiting for title (attempt 2): Just a moment...
2023-09-16 11:42:07 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:07 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:07 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:07 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:07 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:09 DEBUG    ReqId 12592 Waiting for title (attempt 3): Just a moment...
2023-09-16 11:42:10 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:10 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:11 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:11 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:11 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:13 DEBUG    ReqId 12592 Waiting for title (attempt 4): Just a moment...
2023-09-16 11:42:14 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:14 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:14 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:14 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:14 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:16 DEBUG    ReqId 12592 Waiting for title (attempt 5): Just a moment...
2023-09-16 11:42:18 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:18 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:18 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:18 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:18 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:20 DEBUG    ReqId 12592 Waiting for title (attempt 6): Just a moment...
2023-09-16 11:42:21 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:21 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:21 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:21 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:21 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:23 DEBUG    ReqId 12592 Waiting for title (attempt 7): Just a moment...
2023-09-16 11:42:25 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:25 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:25 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:25 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:25 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:27 DEBUG    ReqId 12592 Waiting for title (attempt 8): Just a moment...
2023-09-16 11:42:28 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:28 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:28 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:28 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:28 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:30 DEBUG    ReqId 12592 Waiting for title (attempt 9): Just a moment...
2023-09-16 11:42:31 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:31 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:31 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:31 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:31 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:33 DEBUG    ReqId 12592 Waiting for title (attempt 10): Just a moment...
2023-09-16 11:42:34 DEBUG    ReqId 12592 Timeout waiting for selector
2023-09-16 11:42:34 DEBUG    ReqId 12592 Try to find the Cloudflare verify checkbox...
2023-09-16 11:42:34 DEBUG    ReqId 12592 Cloudflare verify checkbox not found on the page.
2023-09-16 11:42:34 DEBUG    ReqId 12592 Try to find the Cloudflare 'Verify you are human' button...
2023-09-16 11:42:34 DEBUG    ReqId 12592 The Cloudflare 'Verify you are human' button not found on the page.
2023-09-16 11:42:36 DEBUG    ReqId 12592 Waiting for title (attempt 11): Just a moment...
```

Any possible workarounds?

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/releases/v3.3.6
